pkg/controller/kubelet-config/helpers.go

```diff
@@ -170,8 +170,8 @@ func validateUserKubeletConfig(cfg *mcfgv1.KubeletConfig) error {
 				if err != nil {
 					return fmt.Errorf("KubeletConfiguration: invalid value specified for %s reservation in kubeReserved, %s", rr.String(), val)
 				}
-				if q.Sign() == -1 {
-					return fmt.Errorf("KubeletConfiguration: %s reservation value cannot be negative in kubeReserved", rr.String())
+				if q.Sign() < 0 || q.IsZero() {
```

Comment:
Do we want reserved value input > 0 ? 
Also see q.Sign() doc, return value of Sign() will be either -1, 0 or 1 https://github.com/kubernetes/apimachinery/blob/master/pkg/api/resource/quantity.go#L522

Reply:
Yes, we want the reserved value input > 0. 

I tried to mirror how the kubelet validates these parameters, 

https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/eviction/helpers.go#L221-L223

```golang
if quantity.Sign() < 0 || quantity.IsZero() {
	return nil, fmt.Errorf("eviction threshold %v must be positive: %s", signal, &quantity)
}
```